src/Sinch/SinchClient.cs

```diff
@@ -170,5 +202,25 @@ private static string GetSmsRegion(SmsRegion smsRegion)
 
         /// <inheritdoc/>
         public IAuth Auth { get; }
+
+        /// <inheritdoc/>
+        public ISinchVerificationClient Verification(string appKey, string appSecret)
+        {
+            if (string.IsNullOrEmpty(appKey))
+            {
+                throw new ArgumentNullException(nameof(appKey), "The value should be present");
+            }
+            if (string.IsNullOrEmpty(appSecret))
+            {
+                throw new ArgumentNullException(nameof(appSecret), "The value should be present");
+            }
+            
+            var basicAuth = new BasicAuth(appKey, appSecret);
+            // TODO: implement application signed authentication, create IHttp just before the request with SignedRequestAuth
+            var http = new Http(basicAuth, _httpClient, _loggerFactory?.Create<Http>(), JsonNamingPolicy.CamelCase);
+            return new SinchVerificationClient(appKey, appSecret, 
+                new Uri("https://verification.api.sinch.com/"), 
```

Comment:
Do you now if we have other URLs for test systems for instance? And if customers are using them during their development phase?

Reply:
I will have `internal` constructor for passing `baseUrl`, and have no intention to provide possibility to overwrite that in public api

Reply:
Ok, so we assume that customers will always use the production environment, right?

Reply:
Hum, I'm confused now between your 2 statements
 - "have no intention to provide possibility to overwrite that in public api"
 - "they can overwrite host"
 What should they do to target a different host then?

Reply:
Overwrite host on OS level, e.g. https://www.makeuseof.com/tag/modify-manage-hosts-file-linux/
Or, if user fancy, he can try to call `internal` method. 
By public api I mean there is no `public` constructor for `SinchClient` which can allow overwrite of the `baseUrl`
Nevertheless, I think this discussion is already out of scope of this PR.